sensor/common/networkflow/manager/manager_impl.go

```diff
@@ -408,6 +408,13 @@ func (m *networkFlowManager) enrichAndSend() {
 	updatedEndpoints := m.updateComputer.ComputeUpdatedEndpoints(currentEndpoints)
 	updatedProcesses := m.updateComputer.ComputeUpdatedProcesses(currentProcesses)
 
+	flowMetrics.NumUpdatesSentToCentralCounter.WithLabelValues("connections").Add(float64(len(updatedConns)))
+	flowMetrics.NumUpdatesSentToCentralGauge.WithLabelValues("connections").Add(float64(len(updatedConns)))
+	flowMetrics.NumUpdatesSentToCentralCounter.WithLabelValues("endpoints").Add(float64(len(updatedEndpoints)))
+	flowMetrics.NumUpdatesSentToCentralGauge.WithLabelValues("endpoints").Add(float64(len(updatedEndpoints)))
+	flowMetrics.NumUpdatesSentToCentralCounter.WithLabelValues("processes").Add(float64(len(updatedProcesses)))
+	flowMetrics.NumUpdatesSentToCentralGauge.WithLabelValues("processes").Add(float64(len(updatedProcesses)))
```

Comment:
Should the gauge metrics be incremented or set ? I have the feeling that otherwise the counter and gauge metrics would report the same.

Reply:
[829c138](https://github.com/stackrox/stackrox/pull/16624/commits/829c13872affdba283c714326e9ce82b9290bd63)